modules/images-imagestream-import-import-mode.adoc

```diff
@@ -40,4 +42,12 @@ $ oc import-image <multiarch-image-stream-tag>  --from=<registry>/<project_name>
 [NOTE]
 ====
 The `--import-mode=` default value is `Legacy`. Excluding this value, or failing to specify either `Legacy` or `PreserveOriginal`, imports a single sub-manifest. An invalid import mode returns the following error: `error: valid ImportMode values are Legacy or PreserveOriginal`.
-====
\ No newline at end of file
+====
+
+[discrete]
+[id="images-imagestream-import-import-mode-limitations"]
+== Limitations
+
+Working with manifest lists has the following limitations:
+
+* In some cases, users might want to use sub-manifests directly. `oc adm prune images` and the `CronJob` pruner cannot detect when a sub-manifest list is used. As a result, an administrator using `oc adm prune images`, or the `CronJob` pruner, might delete entire manifest lists, including sub-manifests.
```

Comment:
I may not be understanding this correctly, but is the first sentence the solution and the second and third sentence are actually the limitations? You might want to put the first sentence last, if so. (disregard if not applicable)

Reply:
The first sentence is the situation (using sub-manifests). The second and third sentence are the limitations when using sub-manifests directly.